vimeo/PlayerKit
# AF-311: Updates PlayerKit to support CircleCI 2.0

### Ticket
[AF-311](https://vimean.atlassian.net/browse/AF-311)

### Pull Request Checklist

### Ticket Summary
Building on our updated CircleCI 2.0 config from VimeoNetworking we want to update PlayerKit so that it can also build against Circle CI 2.0.

### Implementation Summary
**.circleci/config.yml**: For detail about what each line of the new config does check out the VimeoNetworking PR: https://github.com/vimeo/VimeoNetworking/pull/257. This PR also tells CircleCI to build against Xcode 10.

**CocoaPods**: Resolved two warnings, one related to overriding `ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES` and another related to having not specified a platform verion.

**Fastlane**: Updated our Fastfile to build against Xcode 10.

### How to Test
- The build job for this branch can be found [here](https://circleci.com/gh/vimeo/PlayerKit/78). Although PlayerKit doesn't currently have any tests the build should not be failing.
